docs/production-deployment/cloud/export.mdx

```diff
@@ -178,15 +87,15 @@ Once you've finalized the setup, here's how to monitor the export progress:
    - **Schedule**: The Export job is scheduled to run on an hourly basis, starting at 10 minutes past each hour.
    - **Duration**: The time taken for the export process can vary, so it might not be instantaneous.
 
-2. **Checking the S3 Bucket**:
+2. **Checking the Object Storage**:
 
-   - **Files Arrival**: Post the initial hour of setting up, inspect your S3 bucket.
+   - **Files Arrival**: Post the initial hour of setting up, inspect your object storage.
      You should see the exported Workflow History files.
      These files encapsulate the relevant data from the Closed Workflows.
    - **Directory Structure**: Your exported files will adhere to the following naming convention and path:
 
-   ```command
-   s3://[bucket-name]/temporal-workflow-history/export/[Namespace]/[Year]/[Month]/[Day]/[Hour]/[Minute]/
+   ```bash
```

Comment:
This is not bash though, maybe just don't specify any language?

Reply:
Fixed